src/QueueReceiver.Core/Services/SyncService.cs

```diff
@@ -66,12 +66,18 @@ public async Task StartAccessSync(List<string> plantList, bool removeUserAccess,
                     pcsPersonOidList.Remove(oidException);
                 }
 
+                if (plant.IsVoided == "Y" || string.IsNullOrEmpty(plant.AffiliateGroupId) || string.IsNullOrEmpty(plant.InternalGroupId))
```

Comment:
Move this `if` up as the first statement in the foreach.

Reply:
Done